eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -11,4 +11,10 @@ steps:
       workingDirectory: $(Build.SourcesDirectory)/${{ parameters.Directory }}
       filePath: eng/common/scripts/Verify-Links.ps1 
       arguments: >
-        -urls $(dir -r -i *.md) -rootUrl "file://$(Build.SourcesDirectory)/${{ parameters.Directory }}" -recursive:$false -ignoreLinksFile ${{ parameters.IgnoreLinksFile }}
+        -urls $(dir -r -i *.md) 
+        -rootUrl "file://$(Build.SourcesDirectory)/${{ parameters.Directory }}" 
+        -recursive $false 
+        -ignoreLinksFile ${{ parameters.IgnoreLinksFile }}
+        -branchReplaceRegex "($(System.PullRequest.sourceRepositoryUri)/blob/)(master)(/.*)"
```

Comment:
I wouldn't put parens around master so the replace is just easy `${1}$branchReplacementName${2}`